src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -62,6 +67,7 @@ List<InternalXrefSpec> LoadXrefSpecs(JsonSchema schema, JToken node)
 
                         break;
                     case JArray array:
+
```

Comment:
remove blank line

Reply:
we leave a blank line under `case` habitually

Reply:
do we? I barely see that

Reply:
removed, follow the existing style.